changesets2CSV/changesets2CSV

```diff
@@ -194,6 +565,22 @@ def changeset_csv(output_file, changesets, name=None, summary=False):
             modify_count = 0
             delete_count = 0
             discussion_count = 0
+
+            #Count additions, changes, and deletions that are names
+            changeset_ids = []
+            for set in changesets:
```

Comment:
This can be done with list comprehension:
`changeset_ids = [ set.get('id') for set in changesets ]`